pypesto/petab/importer.py

```diff
@@ -447,6 +447,9 @@ def create_objective(
         Returns
         -------
         A :class:`pypesto.objective.AmiciObjective` for the model and the data.
+        This object is expected to be passed to
+        :class:`PetabImporter.create_problem` to correctly handle fixed
+        parameters.
```

Comment:
Should we have a test to check the expected behavior if one does not pass it to `PetabImporter.create_problem`?

Reply:
The problem is that we don't know the expected behaviour if one does not pass it to `PetabImporter.create_problem`